### PR TITLE
Migrate deployment method from the blog repo

### PR DESCRIPTION
**Pull request by pavelgrinchenko**
It migrates deployment method from the blog repo, but also allows to start server locally (for debugging purposes)

### CODE REVIEW

File: scripts/s3-utils.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const { s3bucket, s3prefix } = require('../src/server/config')
```
Comment (fabiosantoscode) [inline scripts/s3-utils.js:3]: Typo: `s3Bucket` and `s3Prefix`

File: scripts/s3-utils.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  region: AWS_REGION,
+  accessKeyId: AWS_ACCESS_KEY_ID,
+  secretAccessKey: AWS_SECRET_ACCESS_KEY
+})
+
+console.log({
+  AWS_REGION,
+  HEROKU_APP_NAME,
+  s3bucket,
+  s3prefix,
```
Comment (fabiosantoscode) [inline scripts/s3-utils.js:24]: Typos: `s3Bucket` and `s3Prefix`

File: scripts/s3-utils.js (hunk 1 above)
Comment (pavelgrinchenko) [reply scripts/s3-utils.js]: Fixed

**Comment by pavelgrinchenko** [comment]
@fabiosantoscode I'm not sure that ESLint can check it. Do you know such rule?

**Comment by fabiosantoscode** [comment]
This here rule checks for usage of undefined variables: https://eslint.org/docs/rules/no-undef#top

File: scripts/s3-utils.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const { s3Prefix, s3Bucket } = require('../src/server/s3')
```
Comment (fabiosantoscode) [inline scripts/s3-utils.js:3]: This file is now in src/server/utils.js it seems.

**Comment by JIoJIaJIu** [comment]
I think we need to get rid off express server as a backend, SSR and custom scripts for deployment. So deployment method will be based on [CI orb](https://circleci.com/orbs/registry/orb/circleci/aws-s3)

File: scripts/s3-utils.js (hunk 3 above)
Comment (pavelgrinchenko) [reply scripts/s3-utils.js]: `src/server/config.js`. Fixed...

**Comment by fabiosantoscode** [comment]
@JIoJIaJIu due to the long-expiring cache headers, the express server only serves the requests once, and then cloudflare handles revalidation from clients.

File: src/server/middleware/redirects/index.js [hunk 4]
```diff
@@ -1,26 +1,30 @@
...
-const dev = process.env.NODE_ENV !== 'production'
+const OLD_BLOG_URL_REGEXP = /dataversioncontrol\.com/
 
 module.exports = (req, res, next) => {
   const parsedUrl = parse(req.url, true)
-  const { pathname, query } = parsedUrl
   const host = req.headers.host
+  let pathname = parsedUrl.pathname
+
+  // Remove invisible emoji from URL. It's was on old medium blog
```
Comment (shcheklein) [inline src/server/middleware/redirects/index.js:13]: minor: on -> an

Comment (pavelgrinchenko) [reply src/server/middleware/redirects/index.js]: Fixed

**Pull request merged**